subworkflows/local/eggnog.nf

```diff
@@ -20,7 +20,7 @@ workflow EGGNOG {
 
         if ( ! eggnogfile.exists() ) {
             EGGNOG_DOWNLOAD(Channel.fromPath(eggnog_dbpath))
-            ch_dbpath = EGGNOG_DOWNLOAD.out.db
+            ch_dbpath = EGGNOG_DOWNLOAD.out.db_ready
```

Comment:
So, this channel is not emitted right away but only when the download program finishes? In a way, the old name of the corresponding channel, i.e. `db`, would have been more conservative and this change wouldn't be required.

Reply:
Yes. we can keep the old name (already updated)